action.yml

```diff
@@ -22,6 +22,9 @@ inputs:
     description: "A prefix to the tag name (default: `v`)."
     required: false
     default: "v"
+  custom_tag:
+    description: "Custom tag name. If specified, it overrides bump settings."
+    required: true
```

Comment:
@mathieudutour I didn't properly deal with this: I assume `custom_tag` should **not** be required, shouldn't it?

Reply:
Fixed by c5a62ea.